7_child_vaccination.do

```diff
@@ -89,8 +89,17 @@
 			replace c_measles = . if ((inrange(im5,1,3) & (inrange(im6m1y,6667,9999) | inrange(im6m1d,97,98))) | (im11 == 1 & inlist(im26,8,9)) | (inlist(im2,8,9) & inlist(im11,8,9)))	// missing if measles DK/missing for card and memory
 			replace c_measles = . if ((inrange(im5,1,3) & (inrange(im6m2y,6667,9999) | inrange(im6m2d,97,98))) | (im11 == 1 & inlist(im26,8,9)) | (inlist(im2,8,9) & inlist(im11,8,9)))	// missing if measles DK/missing for card and memory
 		}
-	   
-	   
+		if inlist(country_name,"Serbia2019") {
```

Comment:
Just to confirm that you only brought updates in Main back to Nepal branch? Or changes are also enforced onto Serbia and Algeria?

Reply:
> @jianingwwww
> Just to confirm that you only brought updates in Main back to Nepal branch? Or changes are also enforced onto Serbia and Algeria?

As far as I'm concerned and intended, this should only bring updates onto Nepal~

Reply:
Brilliant!!!
Cheers!